### PR TITLE
Add future versions to openshift_facts

### PR DESCRIPTION
**Pull request by smarterclayton**
Allows dev releases to be used

### CODE REVIEW

**Comment by abutcher** [comment]
aos-ci-test

File: roles/openshift_master_facts/test/openshift_master_facts_default_predicates_tests.py [hunk 1]
```diff
@@ -66,6 +66,10 @@
     ('3.3', 'openshift-enterprise', DEFAULT_PREDICATES_1_3),
     ('1.4', 'origin', DEFAULT_PREDICATES_1_4),
     ('3.4', 'openshift-enterprise', DEFAULT_PREDICATES_1_4)
```
Comment (abutcher) [inline roles/openshift_master_facts/test/openshift_master_facts_default_predicates_tests.py]: Missing a comma after this tuple.

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

**Comment by smarterclayton** [comment]
Did the last one actually pass?  Can't tell.

**Pull request merged**